apm-agent-plugins/apm-spring-webflux/apm-spring-webflux-plugin/src/main/java/co/elastic/apm/agent/springwebflux/WebfluxHelper.java

```diff
@@ -270,8 +269,12 @@ private static void fillRequest(Transaction transaction, ServerWebExchange excha
 
     private static void fillResponse(Transaction transaction, ServerWebExchange exchange) {
         ServerHttpResponse serverResponse = exchange.getResponse();
-        HttpStatus statusCode = serverResponse.getStatusCode();
-        int status = statusCode != null ? statusCode.value() : 200;
+        int status = 0;
+        try {
+            status = SpringWebVersionUtils.getStatusCode(serverResponse);
+        } catch (Exception e) {
+            // todo : log
```

Comment:
when is this log to be implemented?

Reply:
Should be now, thanks!
I will do that with other post-review changes.